extensions/amp-story-shopping/0.1/amp-story-shopping-tag.js

```diff
@@ -75,22 +76,88 @@ export class AmpStoryShoppingTag extends AMP.BaseElement {
     return layout === Layout_Enum.CONTAINER;
   }
 
+  /**
+   * @param {string} textContent
+   * @return {boolean}
+   * @private
+   */
+  isRTLLanguage_(textContent) {
```

Comment:
Sent you a message offline about how to implement RTL support!

Reply:
I'll take a look at it and integrate it into this solution once some of the other PRs are in

Reply:
RTL Support was Added with this PR: #37249